libs/features/cdt/cdt.js

```diff
@@ -0,0 +1,118 @@
+import { getMetadata, getConfig, createTag } from '../../utils/utils.js';
+import { replaceKey } from '../placeholders.js';
+
+function loadCountdownTimer(
+  container,
+  cdtLabel,
+  cdtDays,
+  cdtHours,
+  cdtMins,
+  timeRangesEpoch,
+) {
+  let isVisible = false;
+  let interval;
+
+  function appendTimerBox(parent, value, label) {
+    const fragment = createTag('div', { class: 'timer-fragment' }, null, { parent });
+    const unitContainer = createTag('div', { class: 'timer-unit-container' }, null, { parent: fragment });
+    createTag('div', { class: 'timer-unit-label' }, label, { parent: fragment });
+
+    createTag('div', { class: 'timer-box' }, Math.floor(value / 10).toString(), { parent: unitContainer });
+    createTag('div', { class: 'timer-box' }, (value % 10).toString(), { parent: unitContainer });
+  }
+
+  function appendSeparator(parent) {
+    createTag('div', { class: 'timer-separator' }, ':', { parent });
+  }
+
+  function appendTimerBlock(parent, daysLeft, hoursLeft, minutesLeft) {
+    const timerBlock = createTag('div', { class: 'timer-block' }, null, { parent });
+    appendTimerBox(timerBlock, daysLeft, cdtDays);
+    appendSeparator(timerBlock);
+    appendTimerBox(timerBlock, hoursLeft, cdtHours);
+    appendSeparator(timerBlock);
+    appendTimerBox(timerBlock, minutesLeft, cdtMins);
+  }
+
+  function appendTimerLabel(parent, label) {
+    createTag('div', { class: 'timer-label' }, label, { parent });
+  }
+
+  function removeCountdown() {
+    container.innerHTML = '';
```

Comment:
it's more efficient and cleaner if you call `.remove()` on the html elements you want to remove.